form_jsons/public/organisation-information.json

```diff
@@ -839,10 +830,10 @@
         }
       },
       {
-        "displayName": "Co-operative, such as a community benefit society",
-        "name": "MomaFn",
+        "displayName": "Type of organisation-  option(two, three, four)",
+        "name": "mNExjy",
```

Comment:
Does this name need to change?

Reply:
No - they were different elements and not related but the diff is putting them next to each other I think coz of line numbers